LICENSE-binary

```diff
@@ -501,7 +501,6 @@ core/src/main/resources/org/apache/spark/ui/static/d3.min.js
 
 Common Development and Distribution License (CDDL) 1.0
 ------------------------------------------------------
-javax.activation:activation  http://www.oracle.com/technetwork/java/javase/tech/index-jsp-138795.html
```

Comment:
@cnauroth  Here it is. Maybe you didn't notice the change here.

Reply:
Thank you, I do see your change there, but it seems there is one more mention in LICENSE-binary at the root of the repo that could be removed:

https://github.com/apache/spark/blob/master/LICENSE-binary#L504

Reply:
This is the line 504 of `LICENSE-binary`, isn't it, @cnauroth ?

Reply:
Sorry all, misread this as coming from a different file. Thanks for the patch, @wayneguow .